murdock/tests/test_murdock.py

```diff
@@ -717,7 +717,7 @@ async def test_handle_push_event_ref_not_handled(
     fetch_config.assert_called_with(commit)
     fetch_commit.assert_called_with(commit)
     queued.assert_not_called()
-    assert f"Ref '{ref_name}' not accepted for push events" not in caplog.text
+    assert f"Ref '{ref_name}' not accepted for push events" in caplog.text
```

Comment:
Can you try to add `caplog.set_level(logging.DEBUG, logger="murdock")` at the beginning of this test function ?

Reply:
Works!